bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -526,54 +527,63 @@ Warnings:
+| signal              | value             | Number:Power | Read-only  | Signal strength in dBm                                                                                                                      |
+| last-event-doorbell | type              | String       | Read-only  | Type of event                                                                                                                               |
+| last-event-doorbell | video-status      | String       | Read-only  | Status of the video (recording, deleted or available)                                                                                       |
+| last-event-doorbell | time              | DateTime     | Read-only  | Time of occurrence of event                                                                                                                 |
+| last-event-doorbell | local-video-url   | String       | Read-only  | If the last event (depending upon event type) in the home lead a snapshot picture, the corresponding local video URL will be available here |
+| last-event-doorbell | vpn-video-url     | String       | Read-only  | If the last event (depending upon event type) in the home lead a snapshot picture, the corresponding VPN video URL will be available here   |
+| sub-event-doorbell  | type              | String       | Read-only  | Type of sub-event                                                                                                                           |
+| sub-event-doorbell  | time              | DateTime     | Read-only  | Time of occurrence of sub-event                                                                                                             |
+| sub-event-doorbell  | message           | String       | Read-only  | Message sent by Netatmo corresponding to given sub-event                                                                                    |
+| sub-event-doorbell  | snapshot-url      | String       | Read-only  | Depending upon event type in the home, a snapshot picture of the corresponding local video URL will be available here                       |
+| sub-event-doorbell  | vignette-url      | String       | Read-only  | A vignette representing the snapshot                                                                                                        |
+| sub-event-doorbell  | snapshot          | Image        | Read-only  | picture of the snapshot                                                                                                                     |
+| sub-event-doorbell  | vignet            | Image        | Read-only  | picture of the vignette                                                                                                                     |
```

Comment:
Please fix the channel id, it is `vignette`

Reply:
fixed